### ISSUE TITLE
Minor UI: Bids and Asks in marketplace too discrete

### ISSUE THREAD

**Issue by johnmark13** [vessenes/persona#20]
![image](https://github.com/user-attachments/assets/9deefe79-de4a-454c-b4bb-a7d67595600d)

Especially with the glowing folk....

**Comment by vessenes** [comment]
Yes, this is a tough set of tradeoffs. A better designer will eventually make it better. For now, we are going to add a little bit of height to the cards when they render market information, and that will at least back them off the glow.
